server/src/authorizers/channels.spec.js

```diff
@@ -0,0 +1,49 @@
+const channels = require('./channels');
+
+describe('tests channel authorizers', () => {
```

Comment:
This is better. 
A few things though:
1) Don't use the word "tests" in tests. It's obvious it's a test. Instead, just have things like ``Channel Authorizers``
2) Don't use the word "should" in tests. Just phrase them like, "returns true if channel is public"
3) Use other describe blocks for context. For instance, ``describe(`'when the channel is public`)`` and have tests that describe states where the channel is public.

Reply:
Good point, thanks.